### PR TITLE
add description of PM role to contributing doc

### PR DESCRIPTION
**Pull request by natasha41575**
/cc @annasong20

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -64,6 +64,33 @@ Administrative notes:
...
 
+## Project/Product Managers
+
+Kustomize will have opportunities to join in a project/product manager role. You can
+typically start working on this role as part of a kustomize training cohort, so please
+keep an eye out for that or reach out to the leads if you are interested in this type of
+work. Expectations for this role are:
+
+- Triage 1 feature request each week, and bring it to weekly stand-up for discussion. Feature
+requests are issues labeled kind/feature, and you can find them [here](https://github.com/kubernetes-sigs/kustomize/issues?q=is%3Aissue+is%3Aopen+kind+feature+label%3Akind%2Ffeature).
```
Comment (annasong20) [inline CONTRIBUTING.md:75]: No need to fix. Just FYI that https://github.com/kubernetes-sigs/kustomize/labels/kind%2Ffeature also works.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -64,6 +64,33 @@ Administrative notes:
 - The [OWNERS file spec] is a useful resources in making changes.
 - Maintainers and admins must be added to the appropriate lists in both [Kustomize OWNERS_ALIASES] and [SIG-CLI Teams]. If this isn't done, the individual in question will lack either PR approval rights (Kustomize list) or the appropriate Github repository permissions (community list).
 
+## Project/Product Managers
+
+Kustomize will have opportunities to join in a project/product manager role. You can
+typically start working on this role as part of a kustomize training cohort, so please
+keep an eye out for that or reach out to the leads if you are interested in this type of
+work. Expectations for this role are:
```
Comment (annasong20) [inline CONTRIBUTING.md:72]: Non-blocking: Could the PM also be responsible for building or at least documenting the order for some sort of PR or issue-tackling queue? I'm frequently concerned that I missed an email or have been ignoring an old contribution.

**Review by annasong20 — COMMENTED** [review]
/hold for Natasha's review
/lgtm

File: CONTRIBUTING.md (hunk 2 above)
Comment (natasha41575) [reply CONTRIBUTING.md:72]: The goal if I understand you correctly is that they should be helping us easily discover if an old issue or PR has been waiting for us for a long time.

That sounds good to me, I'll add that.

File: CONTRIBUTING.md (hunk 1 above)
Comment (natasha41575) [reply CONTRIBUTING.md:75]: TIL. Thanks!

File: CONTRIBUTING.md (hunk 2 above)
Comment (annasong20) [reply CONTRIBUTING.md:72]: Looks great! Thank you.

**Review by annasong20 — COMMENTED** [review]
/unhold

**Pull request merged**